### ISSUE TITLE
[@types/imapflow] missing field on download method options

### ISSUE THREAD

**Issue by Akstasy** [DefinitelyTyped/DefinitelyTyped#59239]
Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂

Today I used [patch-package](https://github.com/ds300/patch-package) to patch `@types/imapflow@1.0.6` for the project I'm working on.

chunksize option is missing based on documentation:
https://imapflow.com/module-imapflow-ImapFlow.html#connect-examples

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/@types/imapflow/index.d.ts b/node_modules/@types/imapflow/index.d.ts
index d4c0efe..a0ed375 100755
--- a/node_modules/@types/imapflow/index.d.ts
+++ b/node_modules/@types/imapflow/index.d.ts
@@ -36,7 +36,7 @@ export class ImapFlow extends EventEmitter {
     download(
         range: SequenceString,
         part?: string,
-        options?: { uid?: boolean; maxBytes?: number },
+        options?: { uid?: boolean; maxBytes?: number, chunkSize?: number },
     ): Promise<DownloadObject>;
 
     getMailboxLock(path: string, options?: null | { readonly?: boolean }): Promise<MailboxLockObject>;
```

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by Grayson-code** [comment]
fixed in #59241

**Comment by Grayson-code** [comment]
fixed here #59242
